atlas/additional-guides/troubleshooting-builds.mdx

```diff
@@ -67,6 +67,22 @@ After the platform pulls your code and runs `npm install`, it will execute the `
   },
 ```
 
+### Make sure the branch and repository you're pulling from exists
+
+If you've deleted your branch or repository, you'll see the following message on the environment details screen:
+<img width="599" alt="image" src="https://user-images.githubusercontent.com/15729608/177279683-d43d93a3-30f7-4bdc-b883-5f79ab158f92.png">
```

Comment:
@ikedracka For these images, we'll need to include them in an assets folder in the headless docs repo for that specific tutorial or guide: https://github.com/wpengine/headless-docs/tree/platform-branch-change/atlas/additional-guides/assets/img/troubleshooting-builds